src/main/java/org/raml/parser/rule/SchemaRule.java

```diff
@@ -164,6 +164,9 @@ else if (node.getValue().matches("\\w.*"))
     private ScalarNode getGlobalSchemaNode(String key)
     {
         GlobalSchemasRule schemasRule = (GlobalSchemasRule) getRootTupleRule().getRuleByFieldName("schemas");
+        if (schemasRule==null){
+        	return null;
+        }
```

Comment:
Looking at how the code is formatted around your addition, I think your code should be formatted like this:
```java
if (schemasRule == null)
{
    return null;
}
```
ie. opening curly brace on new line, spaces around operators and 4 spaces for indentation.

IDEs are able to do this formatting work for you, if configured properly.